uReplicator-Common/src/main/java/com/uber/stream/kafka/mirrormaker/common/core/KafkaBrokerTopicObserver.java

```diff
@@ -49,24 +51,30 @@
       LoggerFactory.getLogger(KafkaBrokerTopicObserver.class);
 
   private static String KAFKA_TOPICS_PATH = "/brokers/topics";
+  private static String KAFKA_INNER_TOPIC = "__consumer_offsets";
```

Comment:
maybe revert this file and have a separate PR for  KafkaBrokerTopicObserver improvement.